docs/using/administering/debugging.rst

```diff
@@ -33,6 +37,78 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* Flocker version,
+* operating system and version,
+* Linux kernel version,
+* Docker version and configuration,
+* node IP addresses,
+* node hostname,
+* recent syslog content,
+* and separate Flocker log files.
+
+.. warning:: The exported log files may contain sensitive information.
+
+Export Logs Using ``flocker-diagnostics``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``flocker-diagnostics`` command can be used to gather this information on Ubuntu 14.04 and CentOS 7.
+
+.. prompt:: bash #
+
+   flocker-diagnostics
+
+``flocker-diagnostics`` will create a ``tar`` archive in the current directory.
+It will print the full path of the archive before it exits.
+
+Exporting logs manually
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Alternatively, the information can be gathered manually using the following commands:
+
+* Flocker version
+
+  .. prompt:: bash #
+
+     flocker-control --version
+
+* Operating system and version:
```

Comment:
This step/command has a colon, while the others in this list do not. I think it looks better to have the colon before each example (rather than removing this one)